### PR TITLE
new block for regional contact section

### LINKED ISSUE #38: Implement regional contact overview
For reference see: https://www.moleculardevices.com/contact

image

### PR DESCRIPTION
**Pull request by kapildhiman**
Please always provide the [GitHub issue(s)](../issues) your PR is for, as well as test URLs where your change can be observed (before and after):

New block for regional contact section on the contact page

Fix #38

Test URLs:
- Before: https://main--moleculardevices--hlxsites.hlx.page/contact
- After: https://regional-contact--moleculardevices--hlxsites.hlx.page/contact

### CODE REVIEW

File: blocks/regional-contacts/regional-contacts.css [hunk 1]
```diff
@@ -0,0 +1,353 @@
...
+    background-position: top center;
+    padding: 80px 0;
+}
+
+.section.regional-contacts-container img {
+    vertical-align: middle;
+}
+
+.section.regional-contacts-container h2 {
+    color: #666;
```
Comment (mhaack) [inline blocks/regional-contacts/regional-contacts.css:43]: Should we use/add a variable here as well?

File: blocks/regional-contacts/regional-contacts.css [hunk 2]
```diff
@@ -0,0 +1,353 @@
...
+.regional-contacts-wrapper .regional-contacts .tab-accordian-wrapper > div:nth-child(odd) > ul:not(:first-child) li {
+    padding: 10px 0;
+    position: relative;
+    display: flex;
+    align-items: center;
+    flex-wrap: wrap;
+}
+
+.regional-contacts-wrapper .regional-contacts .tab-accordian-wrapper > div:nth-child(even) > ul > li:nth-child(3) {
+    font-size: 18px;
```
Comment (mhaack) [inline blocks/regional-contacts/regional-contacts.css:131]: Use a variable here as well.

Comment (kapildhiman) [reply blocks/regional-contacts/regional-contacts.css]: This is done.

File: blocks/regional-contacts/regional-contacts.css [hunk 3]
```diff
@@ -0,0 +1,353 @@
...
+
+.tab-wrapper > a:not(:last-child) {
+  border-right: 2px solid var(--background-color);
+}
+
+/* TAB END */
+
+.section.regional-contacts-container {
+  background-color: var(--background-color);
+  background-image: url('https://www.moleculardevices.com/themes/moldev/images/regional-office-bg.jpg');
```
Comment (mhaack) [inline blocks/regional-contacts/regional-contacts.css:32]: This URL must point to the image on Franklin which you have here in the PR.

Comment (kapildhiman) [reply blocks/regional-contacts/regional-contacts.css]: This has been fixed.

**Pull request merged**